src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -990,14 +990,14 @@ MavlinkReceiver::handle_message_set_actuator_control_target(mavlink_message_t *m
 	    (mavlink_system.compid == set_actuator_control_target.target_component ||
 	     set_actuator_control_target.target_component == 0) &&
 	    values_finite) {
-
+		bool ignore_setpoints = bool(set_actuator_control_target.group_mlx != 2);
```

Comment:
I think a comment about what `group_mlx == 2` means and why it means we should ignore the setpoints would be good. Otherwise, this looks fine to me.

Reply:
I have updated the comments in the following commit